patch.yml

```diff
@@ -13,6 +13,54 @@
 # limitations under the License.
 
 ---
+- name: Play | Set correct RDBMS release and set fact if this is a single node inventory
+  hosts: dbasm
+  gather_facts: false
+  tasks:
+    - name: include defaults
```

Comment:
I don't see code like this anywhere else in the toolkit, yet we seem to be able to apply defaults.  Any idea why?

Reply:
In all other places this is included as part of role via Ansible's meta dependencies, but here we need these variables at the start of play populated before going down into the roles.

Reply:
Outdated now.